Samples/Swift/DaysUntilBirthday/DaysUntilBirthdayForPod.xcodeproj/xcshareddata/xcschemes/DaysUntilBirthday(macOS).xcscheme

```diff
@@ -0,0 +1,78 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Should we be adding a macOS scheme for the Swift project as well?  It looks like we're only adding an iOS scheme there.

Reply:
Added.